pkg/pricer/headerutils/utilities.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package headerutils
+
+import (
+	"encoding/binary"
+	"errors"
+	"github.com/ethersphere/bee/pkg/p2p"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var (
+	// ErrFieldLenth denotes p2p.Header having malformed field length in bytes
+	ErrFieldLenth = errors.New("field length error")
+	// ErrNoIndexHeader denotes p2p.Header lacking specified field
+	ErrNoIndexHeader = errors.New("no index header")
+	// ErrNoTargetHeader denotes p2p.Header lacking specified field
+	ErrNoTargetHeader = errors.New("no target header")
+	// ErrNoPriceHeader denotes p2p.Header lacking specified field
+	ErrNoPriceHeader = errors.New("no price header")
+)
+
+// Headers, utility functions
+
+func MakePricingHeaders(chunkPrice uint64, addr swarm.Address) (p2p.Headers, error) {
+
+	chunkPriceInBytes := make([]byte, 8)
+
+	binary.BigEndian.PutUint64(chunkPriceInBytes, chunkPrice)
+
+	headers := p2p.Headers{
+		"price":  chunkPriceInBytes,
```

Comment:
please extract header names to package-level `const`s, try to avoid hardcoding literals wherever possible